pkg/maps/ctmap/ctmap.go

```diff
@@ -92,6 +92,12 @@ const (
 	metricsDeleted = "deleted"
 )
 
+type NatMap interface {
```

Comment:
Can we move this declaration to `pkg/maps/nat/types.go`?

Reply:
Won't defeat the purpose of having an interface? :smile:

Reply:
Yeah, interfaces should be declared in the package where they're received, like it is here. I think it's fine.